R/get.R

```diff
@@ -526,6 +526,72 @@ get_usage_static <- function(
   return(out)
 }
 
+#' Get usage information for deployed content
+#'
+#' @description
+
+#' Retrieve content hits for all available content on the server. Available
+#' content depends on the user whose API key is in use. Administrator accounts
+#' will receive data for all content on the server. Publishers will receive data
+#' for all content they own or collaborate on.
+#'
+#' If no date-times are provided, all usage data will be returned.
+
+#' @param client A `Connect` R6 client object.
+#' @param from Optional `Date` or date-time (`POSIXct` or `POSIXlt`). Only
+#'   records after this time are returned. If a `Date`, treated as the start of
+#'   that day in the local time zone; if a date-time, used verbatim.
+#' @param to Optional `Date` or date-time (`POSIXct` or `POSIXlt`). Only records
+#'   before this time are returned. If a `Date`, treated as end of that day
+#'   (`23:59:59`) in the local time zone; if a date-time, used verbatim.
+#'
+#' @return A tibble with columns:
+#' * `id`: An identifier for the record.
+#' * `user_guid`: The GUID of logged-in visitors, NA for anonymous.
+#' * `content_guid`: The GUID of the content.
+#' * `timestamp`: The time of the hit as `POSIXct`.
+#' * `path`: The path of the hit. Not recorded for all content types.
+#' * `user_agent`: If available, the user agent string for the hit. Not
+#'   available for all records.
+#'
+#' @details
+#'
+#' The data returned by `get_usage()` includes all content types. For Shiny
+#' content, the `timestamp` indicates the *start* of the Shiny session.
+#' Additional fields for Shiny and non-Shiny are available respectively from
+#' `get_usage_shiny()` and `get_usage_static()`.
+#'
+#' When possible, however, we recommend using `get_usage()` over
+#' `get_usage_static()` or `get_usage_shiny()`, as it will be much faster for
+#' large datasets.
```

Comment:
What does "large datasets" mean here?

Reply:
I mean, not large in the "big data" sense, but like, to fetch 100,000 records on the old API requires 200 requests, which starts to lead to delays of a multiple seconds scaling linearly. Perhaps best to just not describe it?